### PR TITLE
Quick Select Algorithm

### PR DESCRIPTION
**Pull request by harshitaaa8**
## 🛠️ Issue (#1368 )

Quick Select is a selection algorithm to find the k-th smallest element in an unordered list.

Issue no #1368 

## 👨‍💻 Changes proposed
Added Quick Select Algorithm in c, c++, java, python

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
I have checked all the 4 files are giving the desirable results.

## 📷 Screenshots
Screenshot 2023-06-22 183408

### CODE REVIEW

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
All the program files are missing the extension

**Comment by Kumar-laxmi** [comment]
image

@harshitaaa8 Does it mean your PR doesn't follow any of those guidelines

**Review by zugzwang03 — CHANGES_REQUESTED** [review]
name the files properly

**Comment by harshitaaa8** [comment]
I have added the extensions and checked the boxes as it follows the guidelines.

**Review by 072arushi — APPROVED** [review]

File: C++/Searching/Quick Select Algorithm.cpp
Comment (zugzwang03) [inline C++/Searching/Quick Select Algorithm.cpp:1]: don't keep space in file names

Comment (harshitaaa8) [reply C++/Searching/Quick Select Algorithm.cpp:1]: Okay.
Have done the changes.

**Review by Arun9739 — APPROVED** [review]

**Review by zugzwang03 — APPROVED** [review]

**Review by Kumar-laxmi — COMMENTED** [review]
These programs seems to be copied from https://www.geeksforgeeks.org/quickselect-algorithm/

**Comment by harshitaaa8** [comment]
I have referred to the site but I did not mean to add plagiarized code 
I sincerely apologize and will make sure this is not repeated.
I can close this issue and raise a new issue where I will make sure to not repeat my mistake.

**Pull request closed without merging**